packages/vit-s/src/VitessceGrid.js

```diff
@@ -58,6 +59,8 @@ export default function VitessceGrid(props) {
   const classes = useVitessceContainerStyles();
   const titleClasses = useTitleStyles();
 
+  const altText = getAltText(config);
```

Comment:
```suggestion
  const altText = useMemo(() => getAltText(config), [configKey]);
```

(Note that `useMemo` will need to be imported above from React if this suggestion is accepted)